src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int randomIndex = new Random().nextInt(colors.length);
```

Comment:
make random class field, not local variable, that has to be recreated on each method call